<launch>
    <arg name="debug" default="false" />
    <arg unless="$(arg debug)" name="launch_prefix" value="" />
    <arg     if="$(arg debug)" name="launch_prefix"
           value="gdb --ex run --args" /> 
    <arg name="transform" default="0,0,0,1,0,0,0" />
    <param  name="plan_group" value="edo"/>
    <param  name="ctrl_ns" value="edo"/>

    <node name="live_human" pkg="human_collision_objects" type="live_human" respawn="false" output="screen" launch-prefix="$(arg launch_prefix)">
        <param name="workcell_transform" value="$(arg transform)"/>
    </node> 
    <!-- launch-prefix="gdb -ex run  -args"> -->
    <!-- "gdb -.-ex run -.-args"  -->
</launch>
